src/gui/widgets/settingswidget.cpp

```diff
@@ -433,3 +434,10 @@ bool SettingsWidget::usingStylesheet()
     }
     return false;
 }
+
+void SettingsWidget::on_resetStylePushButton_clicked()
+{
+    DEB << "Resetting style to default...";
+    ui->styleComboBox->setCurrentText(QStringLiteral("Fusion"));
```

Comment:
dont we already have a mapping to that string?

Reply:
right! `AStyle::defaultStyle` should do the trick